src/schema/widget/tags.json

```diff
@@ -7,6 +7,29 @@
     "type": {
       "type": "string",
       "const": "tags"
+    },
+    "orderBy": {
+      "type": "string",
+      "default": "name",
+      "description": "Order tags by name or by length.",
+      "nullable": true
+    },
+    "order": {
+      "type": "integer",
```

Comment:
`order_by` and `order` may be merged into one option, like `order_by: -date` or `order_by: name`. See https://github.com/hexojs/hexo-generator-tag